### PR TITLE
fix: windows path fix

### LINKED ISSUE #133: 0.13.0 breaks custom colors
I updated from v0.12.5 to v0.13.0 and now my **custom colors are not working anymore**.

i also tried to change my `windi.config.js` file to use defineConfig but still not working. before i used plain `module.exports`

```
import { defineConfig } from 'windicss/helpers'

export default defineConfig({
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      blue: {
        dark: '#ff00ff',
      },
      green: {
        DEFAULT: '#ffff00'
      }
......
```

Windows x64
"@sveltejs/kit": "next",
"svelte": "^3.37.0",
"svelte-preprocess": "^4.7.0",
"vite": "^2.1.5",
"vite-plugin-windicss": "^0.13.0"

### LINKED ISSUE #134: 0.13.0 Breaks tailwind.config.js
The latest version seems to break the loading of tailwind.config.js.
My project with version 0.13 are not able to load tailwind.config.js. Version 0.12.5 works fine.

**Error messages :** 

> [vite-plugin-windicss] failed to load config "E:\...\tailwind.config.js"

> [vite-plugin-windicss] Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'e:'

**Configuration :**  
```json
package.json
{
  "name": "xxx",
  "version": "0.0.0",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "serve": "vite preview"
  },
  "dependencies": {
    "vue": "^3.0.5",
    "vue-router": "^4.0.6"
  },
  "devDependencies": {
    "@vitejs/plugin-vue": "^1.2.1",
    "@vue/compiler-sfc": "^3.0.5",
    "vite": "^2.1.5",
    "vite-plugin-windicss": "0.13.0"
  }
}
```

```js
tailwind.config.js
module.exports = {
  theme: {},
  plugins: [
    require('windicss/plugin/aspect-ratio'),
    require('windicss/plugin/forms'),
    require('windicss/plugin/typography')
  ]
}
```

OS: Windows 10

**Error resolution :**  
```
yarn remove vite-plugin-windicss
yarn add -D vite-plugin-windicss@0.12.5
```

Then no error anymore.

Thank you for your work on this package btw, it's awesome ;)

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
this fixes issue in windows, where it needs full path information
closes #133
closes #134

### CODE REVIEW

File: packages/plugin-utils/src/resolveOptions.ts [hunk 1]
```diff
@@ -226,7 +227,8 @@ export async function loadConfiguration(options: LoadConfigurationOptions) {
           // hack to prevent `import` get transformed
           // eslint-disable-next-line no-new-func
           const _import = new Function('modulePath', 'return import(modulePath)')
-          resolved = (await _import(configFilePath))?.default || {}
+
+          resolved = (await _import(pathToFileURL(resolve(configFilePath))))?.default || {}
```
Comment (alexanderniebuhr) [inline packages/plugin-utils/src/resolveOptions.ts:231]: not quite sure if we can get this working without `resolve()`
to improve it further.. with resolve it works

**Comment by alexanderniebuhr** [comment]
@antfu just a side note, if that does not work with Darwin or linux, we might have to wrap it with os switches.. but on my tests it works with macOS big sur and windows 10

File: packages/plugin-utils/src/resolveOptions.ts (hunk 1 above)
Comment (antfu) [reply packages/plugin-utils/src/resolveOptions.ts:231]: It should already be resolved, have you tried without it?

Comment (alexanderniebuhr) [reply packages/plugin-utils/src/resolveOptions.ts:231]: I do check that tomorrow again, to make sure... if we get full path like `C://` I think we can drop `resolve`

**Pull request merged**